plugins/linear-genome-view/src/LaunchLinearGenomeView/index.ts

```diff
@@ -61,11 +61,19 @@ export default (pluginManager: PluginManager) => {
           view.setHideHeader(!nav)
         }
         if (highlight !== undefined) {
-          const location = parseLocString(highlight, refName =>
+          const parsedLocString = parseLocString(highlight, refName =>
             isValidRefName(refName, assembly),
           ) as Required<ParsedLocString>
+
+          const location = {
+            ...parsedLocString,
+            assemblyName: assembly,
+            refName:
+              asm.getCanonicalRefName(parsedLocString.refName) ??
```

Comment:
could consider moving this "asm.getCanonicalRefName(parsedLocString.refName) ?? parsedLocString.refName," to time of usage (e.g. in Highlight.tsx, OverviewHighlight.tsx) rather than data loading time. it is a little more laborious, but would handle cases that don't go through the launchlineargenomeview pathway for whatever reason